packages/jwt-crypto/src/index.ts

```diff
@@ -0,0 +1,96 @@
+import { createRemoteJWKSet, jwtVerify } from "jose"
+import { z } from "zod"
+
+export class JwtService {
+  private jwks: ReturnType<typeof createRemoteJWKSet> | null = null
+  private readonly jwksUrl: URL
+
+  public constructor(private readonly issuer: string) {
+    const issuerWithoutTrailingSlash = issuer.replace(/\/$/, "")
+    this.jwksUrl = new URL(`${issuerWithoutTrailingSlash}/.well-known/jwks.json`)
+  }
+
+  /**
+   * Verify a JWT token using the specified issuer's JWKS
+   */
+  public async verify(token: string) {
+    this.jwks ??= createRemoteJWKSet(this.jwksUrl)
+    return jwtVerify(token, this.jwks, {
```

Comment:
https://auth0.com/docs/secure/tokens/access-tokens/validate-access-tokens

This is missing audience validation, it should contain `https://online.ntnu.no` (but we currently only have one api/default audience)
We should have scope-validation but that is a TODO